### PR TITLE
Fixing note on NTAuth store

### PR DESCRIPTION
**Pull request by mapalko**
The NTauth store note should be updated to match the note here: https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-pki

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks like a good update to me.

**Rich Diff view** for better review readability:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9321/files?short_path=028833d#diff-028833d95785da5f189fe5291e31d7ea4e768cfffb98d975320556441e990b0e

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Kateyanne** [comment]
@mapalko could you please resolve the conflicts? Thanks!

**Comment by illfated** [comment]
**This branch has conflicts that must be resolved**
Only those with write access to this repository can merge pull requests.

**Conflicting files**
windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki.md

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki.md [hunk 1]
```diff
@@ -81,13 +81,8 @@ Sign-in a certificate authority or management workstations with _Enterprise Admi
...
-
-The following PowerShell command can be used to check all certificates in the NTAuth store:
-
-```powershell
-Certutil -viewstore -enterprise NTAuth
-```
+> A domain controller's certificate must chain to a certificate in the NTAuth store in Active Directory. By default, online "Enterprise" Active Directory Certificate Authority certificates are added to the NTAuth store at installation time. If you are using a third-party CA, this is not done by default. If the domain controller certificate does not chain to a trusted CA in the NTAuth store, user authentication will fail. 
+> You can view an AD forest's NTAuth store (NTAuthCertificates) using PKIVIEW.MSC from an ADCS CA. Open PKIView.msc, then click the Action menu -> Manage AD Containers. To see all certificates in the NTAuth store run **Certutil -viewstore -enterprise NTAuth**
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki.md:85]: The last sentence appears to be slightly incomplete, at least to a casual reader. May I suggest the following?
```suggestion
> You can view an AD forest's NTAuth store (NTAuthCertificates) using PKIVIEW.MSC from an ADCS CA. Open PKIView.msc, then click the Action menu -> Manage AD Containers. To see all certificates in the NTAuth store, run **Certutil -viewstore -enterprise NTAuth** from the command-line interface (Cmd.exe).
```
Also, adding a missing comma to indicate the sentence clauses. Feel free to modify the ending part.

**Pull request merged**